src/collections/news/2023-05-17-Baby-Bliss-Bot-AI-Techniques.md

```diff
@@ -0,0 +1,136 @@
+</figure>
+
+As can be seen, the generated images have a superficial resemblance to real
+Bliss symbols and it appears that the GAN has detected and learned some features
+or patterns present in the single Bliss symbols. At the present time it is
+unclear what patterns the GAN has learned exactly.  This is an issue for
+potential future explorations of GAN machine learning models.
+
+### StyleGAN3
+
+- Cindy to help here
+
+## CLIP model
+
+CLIP (Contrastive Language–Image Pre-training) comprises a machine learning
+technique in which images, in addition to textual information describing those
+images are used in the training process. This approach has been recieving
+increased attention in the field of computer vision, where previous advances
+have provided high levels of accuracy in tasks such as the recognition and
+classification of objects in photographs and videos. However, it has been argued
+that state-of-the-art approaches in computer vision have various limitations,
+including for example that the systems are often able to make predictions and
+identification over a small number of categories, preventing the ability to
+identify more general and abstract features, not to mentioned the laborious work
+required in the preparation of the data when the systems are required to
+recognize new objects not previously included in the datasets.
+
+The BBB Project has started experimenting with CLIP models in order to explore
+how the use of multimodal (images and textual) data training strategies can be
+applied for a better understanding of the internal composition of Bliss symbols
+and combined words, and eventually provide avenues for the generation of new
+symbols.
+
+For this exploration, we have created an initial annotated dataset using the
+existing official [Bliss vocabulary database](https://www.blissymbolics.org/index.php/symbol-files).
+In this dataset we have used the images of each Bliss symbol and created a set
+of annotations by extracting two fields contained the official Bliss database,
+namely the “gloss”, which includes different words and synonyms and the
+"explanation", which contains a description about how one particular Bliss
+symbol was constructed.
+
+## YOLO
+
+YOLO: "You only look once" is an AI strategy widely applied to real-time object
+detection. YOLO models are currently being used in various tasks such as the
+identification of objects in self-driving cars and even face recognition
+applications. The key goal of YOLO models is to be able to train a system in
+order to identify a given object in any part of an image, and as a result,
+provide information about the position of that object as well as a
+probabilistic approach to the identification of such object in a corpus of
+fixed categories.
+
+<figure style="text-align: center;">
+<img src="https://user-images.githubusercontent.com/999407/249845642-5d96f6c6-3d91-4c6d-bb22-ead7e4b86078.png"
```

Comment:
@agamba This YOLO image of someone's porch needs a better caption.  When I added it, my editor automatically added the alt text from the original `img`: "Screen Shot 2023-06-29 at 11 02 36 AM".  I've used it as a placeholder for now.

Could you add something more meaningful?  Thanks.

Reply:
Image of a porch showing color boxes around the predictions detected by YOLO model. Dog, Cat, Truck

Source: https://pjreddie.com/darknet/yolo/